lib/elliptic/eddsa/key.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var elliptic = require('../../elliptic');
+var utils = elliptic.utils;
+var assert = utils.assert;
+
+function KeyPair(eddsa, params) {
+  this.eddsa = eddsa;
+  this.pubBytes = params.pubBytes;
```

Comment:
Yes, accessors please! :) And lazy computation to, I think this is how it is done for the ECDSA keypair now.